packages/core/tests/unit/models/deliverable-item-test.js

```diff
@@ -15,11 +15,15 @@ module('Unit | Model | deliverable item', function(hooks) {
         tempId
       })
     );
+    const model2 = run(() =>
+    this.owner.lookup('service:store').createRecord('deliverable-item', {
+      id,
+      tempId
+    })
+  );;
 
     assert.equal(model.get('modelId'), tempId, 'getId returns the tempId of the model when id is absent');
-
-    model.set('id', id);
-
-    assert.equal(model.get('modelId'), id, 'getId returns the id when present');
+  
+    assert.equal(model2.get('modelId'), id, 'getId returns the id when present');
```

Comment:
This changes the behavior of the test, is there no easy way to mock `id`?

Reply:
I don't remember why this changed. It seems to pass now though so I'll change it back.

Reply:
Still kinda changes the behavior of the test, originally we
1. Made model with tempId, check it is tempId
2. Set the actual id
3. Check it is now actual id

Now we just check that if we construct a model with a temp id and an id it takes the id

Reply:
In my previous comment I meant I would revert this line to the previous behavior.